docs/kubernetes/AWSIntegration.md

```diff
@@ -0,0 +1,145 @@
+```
+wget https://raw.githubusercontent.com/projectcalico/calico-kubernetes-coreos-demo/ah3-config-update/manifests/calico-etcd.manifest
+```
+Replace all instances of `<PRIVATE_IPV4>` with your master's IP or hostname. Then, place the manifest file in the `/etc/kubernetes/manifests/` directory. 
+```
+sudo mv calico-etcd.manifest /etc/kubernetes/manifests/
+```
+After a short delay, the kubelet on your master will automatically create a container for the new etcd which can be accessed on port 6666 of your master.
+
+## Running `calico-node`
+On each of your AWS Instances, perform the following steps.
+
+### Install calicoctl and calico_kubernetes
+* Download and install the `calicoctl` binary
+```
+wget https://github.com/projectcalico/calico-docker/releases/download/v0.7.0/calicoctl
+chmod +x calicoctl
+sudo mv calicoctl /usr/bin/
+```
+
+* Download and install the plugin binary
+```
+wget https://github.com/projectcalico/calico-kubernetes/releases/download/v0.2.1/calico_kubernetes
+chmod +x calico_kubernetes
+sudo mkdir -p /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/
+sudo mv calico_kubernetes /usr/libexec/kubernetes/kubelet-plugins/net/exec/calico/calico
+```
+
+* Run Calico Node
+```
+sudo modprobe ipip
+sudo ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666 calicoctl node
+```
+Set up an IP pool with IP-in-IP enabled. This is a  [necessary step](https://github.com/projectcalico/calico-docker/blob/20adfd2b7640af9d85c4af76916e043286691452/docs/FAQ.md#can-i-run-calico-in-a-public-cloud-environment) in any public cloud environment.
+> Note: You only need to call `pool add` once per cluster.
+
+```
+sudo ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666 calicoctl pool add 192.168.0.0/16 --ipip --nat-outgoing
+```
+
+## Configure the Kubelet
+To start using the Calico Network Plugin, we will need to modify the existing kubelet process on each of your nodes.
+
+### Authentication for the API Server
+
+In default configurations, the apiserver requires authentication to access its resources. The Calico plugin supports tokens from Kubernetes Service Accounts.
+
+On a fresh cluster, there will be a single default service token for the cluster on your master. You can extract the token with the following command:
+
+```
+TOKEN=$(kubectl describe secret default-token | grep token: | cut -f 2)
+```
+
+For stronger security, you can create a new Service Account specifically for Calico, and use that account's token:
+
+```
+kubectl create -f - <<EOF
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: calico
+EOF
+TOKEN=$(kubectl describe secret calico-token | grep token: | cut -f 2)
+```
+
+### Per node config
+
+First, you will need to create a `network-environment` file with the following contents:
+```
+ETCD_AUTHORITY=<MASTER_PRIVATE_IPV4>:6666
+KUBE_API_ROOT=https://<MASTER_PRIVATE_IPV4>:443/api/v1/
+KUBE_AUTH_TOKEN=<TOKEN>
+CALICO_IPAM=true
+```
+
+If you're not using https on your API Server, use this insecure API path instead:
```

Comment:
This guide is assuming they are running on a cluster configured with `kube-up`, and so they will be using HTTPS.  We can get rid of this comment regarding insecure API.